.github/workflows/coverage.yml

```diff
@@ -1,4 +1,4 @@
-name: CI
```

Comment:
It's named CI to get grouped with the other CI jobs and to be shorter

Reply:
it doesn't seem to get grouped together, at least not in the checks tab, and it just makes it hard to differentiate between them in that window

Reply:
here

![image](https://user-images.githubusercontent.com/552961/99119425-37cf8a80-25c7-11eb-84a7-16316466b05f.png)

Reply:
The coverage jobs are in a separate workflow from the rest of the CI workflow only because the coverage job must also be run on #main so that there is something to diff against when it's run on PRs against main and the GH workflow trigger logic got complex to represent that all in one workflow.

Reply:
![Screenshot from 2020-11-13 13-00-47](https://user-images.githubusercontent.com/1993852/99120807-55452a00-25b0-11eb-8810-4597ec4bdd38.png)

Reply:
They are split up unfortunately because of how GH groups reports from Check jobs, so even if named, 'Coverage' will include spurious reporting about Clippy, etc, yet another reason I kept everything together

![image](https://user-images.githubusercontent.com/552961/99502952-c50a3a80-294b-11eb-919b-49c70346539c.png)